bounties_api/std_bounties/views.py

```diff
@@ -12,7 +12,8 @@
 from std_bounties.filters import BountiesFilter, FulfillmentsFilter
 from rest_framework.filters import OrderingFilter, SearchFilter
 from rest_framework_filters.backends import DjangoFilterBackend
-
+from std_bounties import models
+from django.core import serializers
```

Comment:
@murduk it's not good practice to import the entire set.  Follow the patterns above ^^, and just add the specific serializer and model you need.  With this, things are imported doubly, and this won't pass pep8.